libs/coin-modules/coin-hedera/src/logic/getBalance.test.ts

```diff
@@ -152,16 +162,16 @@ describe("getBalance", () => {
     expect(result[1]).toEqual({
       value: BigInt("5000"),
       asset: {
-        type: "hts",
-        assetReference: "0.0.7890",
+        type: mockToken1.tokenType,
+        assetReference: mockToken1.contractAddress,
         assetOwner: address,
-        name: "Test Token 1",
-        unit: { name: "TT1", code: "tt1", magnitude: 6 },
+        name: mockToken1.name,
+        unit: mockToken1.units[0],
       },
     });
   });
 
-  it("should handle errors from apiClient.getAccount", async () => {
+  it("should throw an error when getAccount throws an error", async () => {
```

Comment:
Maybe rephrase the description to
`should Throw when failing to getAccount data`